### ISSUE TITLE
no binaries are installed

### ISSUE THREAD

**Issue by jbreckel** [singapore/renovate#445]
if I install the module (globally or locally) no link is added into the `.bin` directory.  
should the `bin` field in `package.json` be updated to `bin/renovate.js` from `dist/`?

**Comment by rarkins** [comment]
I'm not an expert on npm binary bundling so will need to research this to know. For which use case do you need this? I thought/hoped it works fine if you `npm i -g renovate` and then run `renovate ...`. Is there another way you are running? e.g. adding it to your `package.json` and hoping to run it out of `./node_modules/.bin/renovate` ?

**Comment by rarkins** [comment]
Here is npm's guide https://docs.npmjs.com/files/package.json#bin

>  On install, npm will symlink that file into prefix/bin for global installs, or ./node_modules/.bin/ for local installs.

That seems to be what I'm already doing, so I'm not sure what to change.

**Comment by jbreckel** [comment]
it is neither added to the global bin directory nor the local one.  
but this may be because of the mismatch of `yarn` and some weird side-effect of `--ignore-engines`. I will try again after the next release.

**Comment by rarkins** [comment]
Uh oh:

```
❯ npm install --save renovate
npm WARN bintest-npm@1.0.0 No description
npm WARN bintest-npm@1.0.0 No repository field.

npm ERR! path /Users/rhys/github/bintest-npm/node_modules/renovate/dist/renovate.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/rhys/github/bintest-npm/node_modules/renovate/dist/renovate.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

**Comment by jbreckel** [comment]
this, looks like it was released without the `dist` folder

**Comment by rarkins** [comment]
Believe it or not, this was caused by a versioning problem. Build process designed for npm@5, CI service uses npm@4 and doesn't look at engines field. Now fixed in 9.6.1:

```
❯ npm install --save renovate@9.6.1
npm WARN bintest-npm@1.0.0 No description
npm WARN bintest-npm@1.0.0 No repository field.

+ renovate@9.6.1
added 13 packages in 3.958s

~/github/bintest-npm
❯ ls node_modules/.bin
bunyan        handlebars    mkdirp        rc            rimraf        showdown      sshpk-sign    uglifyjs      which
changelog     har-validator ncp           renovate      semver        sshpk-conv    sshpk-verify  uuid
```
